style.css

```diff
@@ -22,7 +22,21 @@
 }
 
 /* q */
-.dokuwiki q.blockquote-plugin {
+.dokuwiki q.blockquote-plugin {\0022
```

Comment:
If I leave the **\0022** entity in the CSS code I get an error in DokuWiki ("A fatal error occurred during compilation of the CSS files..."). IMHO that has definitely to be removed. So I did all my tests without that entity.